extension/githubappauthextension/go.mod

```diff
@@ -0,0 +1,54 @@
+module github.com/liatrio/liatrio-otel-collector/extension/githubappauthextension
+
+go 1.22.2
```

Comment:
This should be set to `1.21` with no minor version set to be consistent with other packages and version bumps.